pkg/apis/core/types.go

```diff
@@ -136,8 +136,6 @@ type VolumeSource struct {
 	// AzureDisk represents an Azure Data Disk mount on the host and bind mount to the pod.
 	// +optional
 	AzureDisk *AzureDiskVolumeSource
-	// PhotonPersistentDisk represents a Photon Controller persistent disk attached and mounted on kubelets host machine
-	PhotonPersistentDisk *PhotonPersistentDiskVolumeSource
```

Comment:
I don't think we can actually remove fields from the API.  The best we can do is mark it as deprecated...